src/sonic-ctrmgrd/tests/container_startup_test.py

```diff
@@ -0,0 +1,391 @@
+import unittest
+from unittest.mock import MagicMock, patch
+from . import common_test
+import sys
+sys.path.append("ctrmgr")
+import container_startup
```

Comment:
Suggest reorganizing:

```python
import sys
import unittest
from unittest.mock import MagicMock, patch

from . import common_test

sys.path.append("ctrmgr")
import container_startup
```

And why `import unittest`? Suggest using pytest instead; you're already installing it via `tests_require=` of setup.py.

Reply:
We need unittest.TestCase.
Either we do
      "from unittest import TestCase" and use TestCase or "import unittest" and use unittest.TestCase

Which would you prefer?

Reply:
Why do we need it at all? The tests are run using PyTest, correct? We shouldn't need `unittest.TestCase`

Reply:
Dropped "import unittest" and only import TestCase as supported [here](https://docs.pytest.org/en/stable/unittest.html).

This is what I observe python3 packages that uses class object in TestCode.  
Please let me know, if you would like any further update. A reference would be a big help.

Reply:
Done. Thanks for the reference from Joe!